templates/stripes/stripe-reference--default.php

```diff
@@ -1,14 +1,18 @@
 
-<?php if (!empty($referencePage)) : ?>
+<?php if (!empty($referencePage)): ?>
 <div class="vssl-stripe-column">
     <div class="vssl-stripe--reference--card vssl-stripe--card">
-        <?php if (!empty($referencePage['image'])) : ?>
+        <?php if (!empty($referencePage["image"])): ?>
         <div class="vssl-stripe--reference--image">
-            <a href="<?= $referencePage['slug'] ?>" tabindex="-1">
+            <a href="<?= $referencePage["slug"] ?>" tabindex="-1">
                 <img
                     class="vssl-stripe--reference--thumbnail"
-                    src="<?= $this->image($referencePage['image'], !empty($image_style) ? $image_style : null) ?>"
-                    alt="<?= $referencePage['image'] ?? '' ?>"
+                    src="<?= $this->image(
+                      $referencePage["image"],
+                      !empty($image_style) ? $image_style : null
+                    ) ?>"
+                    alt="<?= $referencePage["image_alt"] ??
```

Comment:
Using new `$image_alt` property on the referenced page for the `alt` attribute of the `<img>` tag, defaulting to the page's image's filepath if not set.